### DISCUSSION TITLE
Historical observation data

### DISCUSSION THREAD

**Discussion by wthueb** [weather-gov/api#671] [Q&A] [answered]
Is the `/stations/{stationId}/observations` endpoint only for observation data up to a week old? Is there no way of accessing observation data from older than a week? I'm polling the DB every day or so, and saving the responses to my own database, but I'd like to get older (hourly) data on startup. If there's another data source, that would work too I guess.

i.e. [`/stations/KFRG/observations?start=2023-10-01T00:00:00Z&end=2023-10-30T00:00:00Z`](https://api.weather.gov/stations/KFRG/observations?start=2023-10-01T00:00:00Z&end=2023-10-30T00:00:00Z) returns no data.

**Comment by akrherz** [comment, marked as answer]
See previous discussions like #646 and #275

weather.gov is restricted by policy to providing a limited archive of data (~7 days).  The authoritative source of archived data is [NCEI](https://www.ncei.noaa.gov) .  Getting data from NCEI is often quite painful and there's many entities providing archived data services, including [Synoptic](https://synopticdata.com/weatherapi/)
